tests/testthat/test_classif_plsdaCaret.R

```diff
@@ -36,3 +36,41 @@ test_that("classif_plsdaCaret", {
   testProbParsets("classif.plsdaCaret", binaryclass.df, binaryclass.target, binaryclass.train.inds,
     old.probs.list, parset.list)
 })
+
```

Comment:
Please remove the unnecessary empty lines below.

Reply:
Removed one line.